### DISCUSSION TITLE
Correct syntax to install imageglass silently and only for certain filetype

### DISCUSSION THREAD

**Discussion by t-leblanc** [d2phap/ImageGlass#1463] [Q&A]
Hello community,

I would like to install ImageGlass silently. I am struggling to find the right syntax for the msi installer. I checked the suggested page https://jrsoftware.org/ishelp/index.php?topic=setupcmdline but that didn't help me much further.

I also only want to register ImageGlass for .heic files for now.

Can someone help me with the correct install syntax through command line?

Thank you in advance!
Thomas

**Comment by d2phap** [comment]
That link is for `.EXE` installer packed with Inno Setup. ImageGlass is packed in MSI installer, you may try this instead: https://stackoverflow.com/questions/8560166/silent-installation-of-a-msi-package

**Reply by d2phap** [reply]
Another link: https://www.advancedinstaller.com/silent-install-exe-msi-applications.html
